api-tests/dev_apis/crypto/test_c002/test_data.h

```diff
@@ -161,7 +161,7 @@ static const uint8_t ec_keypair[] = {
  0x68, 0x49, 0xf9, 0x7d, 0x10, 0x66, 0xf6, 0x99, 0x77, 0x59, 0x63, 0x7c, 0x7e, 0x38,
  0x99, 0x46, 0x4c, 0xee, 0x3e, 0xc7, 0xac, 0x97, 0x06, 0x53, 0xa0, 0xbe, 0x07, 0x42};
 
-static test_data check1[] = {
+static const test_data check1[] = {
```

Comment:
The const qualifier needs to be removed from check1 for the following crypto tests, due to the key_handle member, which is written to by the crypto functions: 2-5, 16, 18-21, 24-44, 46-61 (ranges are inclusive)

(In some cases the const may already be removed, just these were the tests where I saw writes)

Alternatively the key_handle could be moved out of the check1 struct to allow the rest of it to be const.